openstates/oh/bills.py

```diff
@@ -1,5 +1,6 @@
 import os
 import datetime
+from operator import itemgetter
```

Comment:
Hah, I didn't know about this, I usually write `lambda x: x['date']`, but this is fun. Yay `operator`!